src/GrafanaDashboardDeploy/src/DeployTool.cs

```diff
@@ -0,0 +1,152 @@
+using Microsoft.Azure.KeyVault;
+using Microsoft.Azure.Services.AppAuthentication;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace DotNet.Grafana
+{
+    public static class DeployTool
+    {
+        public static async Task PostExportPack(GrafanaClient grafanaClient, string inputFilePath)
+        {
+            var folderIdMap = new Dictionary<string, int>();
+
+            using var sr = new StreamReader(inputFilePath);
+            using var jr = new JsonTextReader(sr);
+            var data = await JObject.LoadAsync(jr).ConfigureAwait(false);
+
+            foreach (var folder in data["folders"])
+            {
+                string folderUid = folder["uid"].Value<string>();
```

Comment:
I think you can say `folder.Value<string>("uid")`, to save an indexer?

Reply:
I tried to stick to indexers all over for consistency.